utils/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'java'
+}
+
+group 'uk.gov.hmcts.reform'
+version '0.0.1'
+
+repositories {
+    mavenCentral()
+}
+
+dependencyCheck {
+  suppressionFile = "${rootDir}/config/owasp/suppressions.xml"
+}
+
+bootJar {
+    enabled = false
+}
```

Comment:
Content of this file is quite different from the other module build.gradle files (e.g. inclusion of repositories element, no jar element etc.).  Is this correct or should it be more like the others?

Reply:
removed repositories and included jar element

Reply:
Funnily enough, resolved in SDT-64

Reply:
Thanks Mark.  For reference the change is in Pull Request https://github.com/hmcts/civil-sdt/pull/28

Reply:
changed it now

Reply:
Mark also removed the group and version (lines 3 and 4) - should these be removed as well?

Reply:
The group and version lines have now been removed.